Dockerfile

```diff
@@ -1,30 +1,29 @@
 # Use the official Golang image to build the binary
 FROM golang:1.20 AS build
-ENV CGO_ENABLED 0
-ARG BINARY_NAME
+ENV CGO_ENABLED = 0
+
 # Set the working directory
 WORKDIR /app
 
 # Copy the Go source files, Makefile, etc.
 COPY . .
 
-# Build the Go application passing BINARY_NAME from Makefile (local development)
-# or Github Action Build-Arg. 
-RUN go build -o ${BINARY_NAME}
```

Comment:
Does this break the makefile though for build? The makefile should be passing in the BINARY_NAME to docker when it is called. Ideally, we do not break the way local development works but enforce it

Reply:
GH Actions wouldn't run without me removing it since it could not recognize {BINARY_NAME} In the ways I tried. I can look around and see how else I can keep the local environment.